src/picologging/logrecord.cxx

```diff
@@ -276,6 +288,21 @@ PyObject* LogRecord_getMessage(LogRecord *self)
     return self->message;
 }
 
+PyObject* LogRecordLogRecord_getnewargs(LogRecord *self)
+{
```

Comment:
Use Py_BuildValue with strong references e.g. `(OOOOOOO)` as the format string instead of `PyTuple_SetItem`

https://docs.python.org/3/c-api/arg.html#c.Py_BuildValue

Reply:
This is probably the cause of the segmentation faults in the tests, because https://docs.python.org/3/c-api/tuple.html#c.PyTuple_SetItem uses borrowed/stolen references to the fields